### ISSUE TITLE
VSCode Bug: Shortcut do not work properly on v1.25.0

### ISSUE THREAD

**Issue by andremacola** [liximomo/vscode-sftp#214]
```
	{
		"key": "cmd+d",
		"command": "sftp.trans.remote",
		"when": "editorFocus"
	},
```

## Expected Behavior
Upload current file on editor focus. Upload from sidebar menu is working fine.

## Actual Behavior
Showing to select a folder (?)

## Specifications
  - Version: 1.25.0-insider using workspace
  - Platform: High Sierra 10.13.5

**Comment by liximomo** [comment]
You must keep the focus on the current file.

**Comment by liximomo** [comment]
Which version of sftp are you using?

**Comment by andremacola** [comment]
@liximomo 1.2.2 version. List all extensions below.

Vscode Insiders latest build.

*Extensions*
Shan.code-settings-sync
anthonydiametrix.ACF-Snippet
bmewburn.vscode-intelephense-client
christian-kohler.path-intellisense
dbaeumer.vscode-eslint
dzannotti.vscode-babel-coloring
emmanuelbeziat.vscode-great-icons
Equinusocio.vsc-material-theme
felixfbecker.php-intellisense
formulahendry.auto-close-tag
liximomo.sftp
ms-vscode.sublime-keybindings
sadesyllas.vscode-workspace-switcher
tungvn.wordpress-snippet
vincaslt.highlight-matching-tag
vsmobile.vscode-react-native

**Comment by liximomo** [comment]
It's likely a bug of Vscode Insiders.

**Comment by andremacola** [comment]
@liximomo now something changed. When I use the keyboard shortcut the sftp is retrieving and transfering all the files.

![captura de tela 2018-06-28 as 00 49 01](https://user-images.githubusercontent.com/3408809/42012135-2006a83c-7a6d-11e8-86a7-0a9e1183eb6d.png)

**Comment by liximomo** [comment]
The Vscode Insider is not stable. I will not handle problems of it.

**Comment by AndisGrossteins** [comment]
@liximomo This issue is reproducible in latest public release of VSCode (1.25.0), so it is not just insider builds now.
I just updated VSCode to 1.25 and I see other extensions throwing errors like `TypeError: Cannot read property 'document' of undefined`.
vscode-sftp doesn't log any errors even with `"sftp.printDebugLog": true` but those error messages might be a good pointer for you.

**Comment by qadszilla** [comment]
This is indeed happening on 1.25

Running "sftp.trans.remote" via keybinding results in all the files being uploaded, I haven't tried "sftp.trans.local" - I have disabled my sftp keybindings for now.

**Comment by liximomo** [comment]
Yes. VSCode 1.25 we can't get the active document.  Wish they can fix this soon.

**Comment by qadszilla** [comment]
Yep, everything works as it should on VS 1.24.1 using sftp 1.2.3; I have downgraded for time being.

**Comment by wilsonchow** [comment]
Hello, just tested keyboard shortcut with sftp 1.2.5 on vscode 1.25.1

macOS 10.13.6 = no problem, upload single file
Windows 7 = ask which local directory and still upload the whole folder files

Like @qadszilla , I have to downgrade back to 1.24.1 on my Windows 7, and works great.

Please help and thanks a lot!

**Comment by liximomo** [comment]
@wilsonchow  we need to wait vscode for this fix.

**Comment by MarekLew** [comment]
Best solution witch actual bechawior, change `keybildings.js`:
```
{
    "key": "ctrl+u",
    "command": "sftp.trans.remote",
    "when": "editorTextFocus",
    "args": [null]
},
```

**Comment by momon** [comment]
Solution proposed by MarekLew above seems to solve it for me at least temporarily.  Thanks!

**Comment by liximomo** [comment]
Fix in v1.3.0.
